data/ABHIGYAN-MOHANTA.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Abhigyan Mohanta",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "| Software Engineer | Java | Python | WebDev (MERN Stack) | Twilio Field Operator | #100Devs |",
+  "avatar": "https://github.com/ABHIGYAN-MOHANTA.png",
```

Comment:
You don't need to add an avatar. It will automatically fetch from GitHub.
```suggestion

```

Reply:
Oh, thanks! Didn't know about that.